kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/KafkaDataBuilder.java

```diff
@@ -72,4 +72,16 @@ public static Map<String, Object> buildKafkaDataRecord(SinkRecord kafkaConnectRe
         return kafkaData;
     }
 
+    /**
+     * Construct a map of Kafka Data record for sending to Storage Write API
+     *
+     * @param kafkaConnectRecord Kafka sink record to build kafka data from.
+     * @return HashMap which contains the values of kafka topic, partition, offset, and insertTime in microseconds.
+     */
+    public static Map<String, Object> buildKafkaDataRecordSorageApi(SinkRecord kafkaConnectRecord) {
```

Comment:
Can we rename it to `buildKafkaDataRecordStorageApi` instead of `buildKafkaDataRecordSorageApi` ?

Reply:
Fixed.